.swiftlint.yml

```diff
@@ -0,0 +1,34 @@
+disabled_rules:
+- trailing_whitespace
+- trailing_comma
+- nesting
+- identifier_name # TODO: this will be re-enabled in the future
+opt_in_rules:
+- empty_count
+- empty_string
+excluded:
+- .build/
+- build/
```

Comment:
@richardpiazza  - This doesn't look right to me.  Are these values (with hyphens) supposed to be indented?

Reply:
Lint ran locally for me with the file, but I'll double check the formatting.

Reply:
With yaml both forms are fine. Both are treated the same and parsed the same in most (not actually all) parsers.

```yaml
parent:
- item1
- item2
```

OR

```yaml
parent
  - item1
  - item2
```